### DISCUSSION TITLE
simplified react-hook-form

### DISCUSSION THREAD

**Discussion by akilansengottaiyan** [react-hook-form/react-hook-form#9180] [Show and tell]
[simplified react hook form](https://github.com/akilansengottaiyan/react-hook-form-simplified) is the simplified version of [react-hook-form@7.34.2](https://github.com/react-hook-form/react-hook-form) intended only for the demonstration purposes.

The goal is to make new contributors or developers who are curious about  react-hook-form get acquainted easily with the library. Look into the [readme](https://github.com/akilansengottaiyan/react-hook-form-simplified#readme) for more about the project.
I hope that it would be helpful for the community.
Comments are most welcome.

**Comment by bluebill1049** [comment]
Awesome stuff, thanks for sharing.
